src/SonarQube.TeamBuild.Integration.Legacy/SonarQube.TeamBuild.Integration.Legacy.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Doesn't _BuildAgentPayload.csproj_ need to be updated to include this new assembly in the zip?

Reply:
That might not be necessary, but I've updated it anyway (in a separate PR).